api/proto/kent/api/session_launch/session_lifecycle.proto

```
@@ -99,10 +100,16 @@ message SessionRetargetWorkspaceRequest {
       expression: "this.trim() != ''"
     }
   ];
+  optional kent.api.worktree.RuntimeStepOrigin origin = 4;
 }
 
 message SessionRetargetWorkspaceSuccess {
-  kent.api.project.ProjectBinding binding = 1 [(buf.validate.field).required = true];
+  oneof result {
+    option (buf.validate.oneof).required = true;
+
+    kent.api.project.ProjectBinding binding = 1;
```

Buf (1.72.0)

[error] Field "1" with name "binding" on message "SessionRetargetWorkspaceSuccess" moved from outside to inside a oneof.

(FIELD_SAME_ONEOF)